extension/gulpfile.babel.js

```diff
@@ -131,7 +146,7 @@ gulp.task('package', function () {
 
 gulp.task('build', (cb) => {
   runSequence(
-    'lint', 'babel', 'chromeManifest',
+    'copy', 'lint', 'babel', 'chromeManifest',
```

Comment:
I think so.  Filed this upstream: https://github.com/yeoman/generator-chrome-extension/issues/158